data/src/main/java/org/openmbee/sdvc/data/domains/Base.java

```diff
@@ -20,7 +20,7 @@
     private static final long serialVersionUID = 8389104517465359723L;
 
     @Id
-    @GeneratedValue(strategy = GenerationType.AUTO)
+    @GeneratedValue(strategy = GenerationType.IDENTITY) //use of AUTO with mysql results in TABLE, for postgres auto is fine
```

Comment:
Was this also tested with postgres? I thought there was an issue with using identity.

Reply:
testing with postgres works, i think it's just less efficient, would be nice if we can use IDENTITY just for mysql